.github/workflows/ci.yml

```diff
@@ -3,27 +3,41 @@ on: pull_request
 
 jobs:
   test:
-    name: build & test
+    name: Build & test
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Checkout source code
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
           node-version: '12'
       - name: Install dependencies
         run: npm ci
-      - run: npm run test:coverage
+      - name: Code coverage
+        run: npm run test:coverage
       - name: Lint
         uses: hallee/eslint-action@1.0.3
         if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }}
         with:
           repo-token: ${{secrets.GITHUB_TOKEN}}
-      - name: Upload coverage report
+      - name: Store code coverage as workflow artifact
         uses: actions/upload-artifact@v2
         with:
-          name: coverage
+          name: istanbul-code-coverage.zip
           path: coverage
-      - name: Code coverage
+      - name: Provide code coverage info as comment in pull request
         uses: codecov/codecov-action@v1
+      - name: Provide sonar static code analysis as comment in pull request
+        uses: sonarsource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          projectBaseDir: .
```

Comment:
Do we need to provide it as `.` because I remember its by default.

Reply:
yes, it's needed to provide `projectBaseDir` if other parameters are provided

Reply:
oks.